data/en/fileiseof.json

```diff
@@ -17,6 +17,22 @@
 	},
 	"links": [
 
+	],
+	"examples": [
+		{
+			"title":"Simple usage syntax",
+			"description":"Call fileIsEOF on a file object and save the result to a variable.",
+			"code":"ileObj = fileOpen(expandPath('./file.txt');\r\nisEndOfFile = fileIsEOF(fileObj);",
```

Comment:
variable name is missing a letter

Reply:
Thank you @KamasamaK . I've fixed the typo in my latest commit.